src/components/auth/node.tsx

```diff
@@ -26,6 +27,34 @@ export function Node(props: NodeProps) {
 
   if (isUiNodeInputAttributes(attributes)) {
     const shortName = attributes.name.replace('traits.', '')
+    const fieldName = hasOwnPropertyTs(strings.auth.fields, shortName)
+      ? (strings.auth.fields[shortName] as string)
+      : shortName
+
+    const messages = node.messages.map(({ text, id }, k) => {
+      const hasTranslatedMessage = hasOwnPropertyTs(strings.auth.messages, id)
+      const rawMessage = hasTranslatedMessage
```

Comment:
maybe we sohuld turn this into a helper / component since we use it in `node` and `flow`.